manifests/init.pp

```diff
@@ -214,5 +216,8 @@
     password_hash         => $grub_password_hash,
     boot_without_password => $boot_without_password,
   }
+  class { 'os_hardening::umask':
```

Comment:
I kind of feel like this should not be a separate module but part of login_defs.

Reply:
login_defs class is clearly made for managing /etc/login.defs, I don't see why it should manage umask, unless I set umask value in login.defs file which is not the case